src/components/common/Button/Button.js

```diff
@@ -3,9 +3,20 @@ import PropTypes from 'prop-types';
 
 import styles from './Button.module.scss';
 
-const Button = ({ children, variant, noHover, className: propClassName, ...props }) => {
+const Button = ({
+  children,
+  favorite,
+  compare,
+  variant,
+  noHover,
+  className: propClassName,
+  ...props
+}) => {
   const classes = [];
 
+  if (favorite) classes.push(styles.active);
```

Comment:
Komentowałem to już ale chyba się nie zapisał komentarz :/ Button jest ogólnym komponentem i nie powinien wiedzieć czy wyświetla jakieś compare czy ulubione czy cokolwiek innego. Tu powinna być tylko flaga np. isActive

Reply:
Poprawione, cała logika jest w ProductBox i tam są dwa propsy isFavorite i isCompare i tam są dodawane klasy zmieniające style Buttonów.